Backend/app/auth/auth_service.py

```diff
@@ -81,20 +78,17 @@ def create_access_token(data: dict[str, str], expires_delta: timedelta | None =
 def get_jwt_token_data(
     token_raw_data: str,
 ) -> TokenData:
-    """Decrypt jwt data and returns data from it
+    """Decodes a JWT token and extracts the user data.
 
     Args:
-    ----
-        token_raw_data (TokenData): JWT Token
-
-    Raises:
-    ------
-        BadJWTTokenProvidedException: invalid token provided
+       token_raw_data (str): Raw JWT token string to decode.
```

Comment:
I think you had a bug here - mismatch between hint in def line (str) vs docstring (TokenData). I assumed the def line is correct version.